op-node/p2p/gating/expiry.go

```diff
@@ -47,19 +46,19 @@ func AddBanExpiry(gater BlockingConnectionGater, store ExpiryStore, log log.Logg
 
 func (g *ExpiryConnectionGater) peerBanExpiryCheck(p peer.ID) (allow bool) {
 	// if the peer is blocked, check if it's time to unblock
-	expiry, err := g.store.PeerBanExpiry(p)
+	expiry, err := g.store.GetPeerBanExpiration(p)
+	if err == store.UnknownBanErr {
```

Comment:
Is there a semantic different from doing an equality check with `==` and using `errors.Is`?

Reply:
I feel like `errors.Is` will catch wrapped errors where an equality check wont?

Reply:
`errors.Is` would work, but similar to `EOF` in many IO stdlib funcs, returning the exact error from `GetPeerBanExpiration` is part of the api here, so it shouldn't make a difference.

Reply:
True, I'm just paranoid. :)